ThunderstoreCLI/Utils/Spinner.cs

```diff
@@ -23,24 +21,37 @@ public ProgressSpinner(string label, Task[] tasks)
 
     public async Task Start()
     {
+        // Cursor operations are not always available e.g. in GitHub Actions environment. Done up here to minimize exception usage (throws and catches are expensive and all)
```

Comment:
That's a quite long line. I can't recall what the enforced line limit should be in this repo, or whether comments are exempt from it, but I'd prefer if this was split to two lines.